src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/Pages/Error.cshtml

```diff
@@ -1,16 +1,16 @@
-﻿@page "/error"
-
-
-<h1 class="text-danger">Error.</h1>
-<h2 class="text-danger">An error occurred while processing your request.</h2>
-
-<h3>Development Mode</h3>
-<p>
-    Swapping to <strong>Development</strong> environment will display more detailed information about the error that occurred.
-</p>
-<p>
-    <strong>The Development environment shouldn't be enabled for deployed applications.</strong>
-    It can result in displaying sensitive information from exceptions to end users.
-    For local debugging, enable the <strong>Development</strong> environment by setting the <strong>ASPNETCORE_ENVIRONMENT</strong> environment variable to <strong>Development</strong>
-    and restarting the app.
-</p>
\ No newline at end of file
+﻿@page
```

Comment:
I don't think the Blazor Server template includes a layout for razor pages. Does the work without it? Doesn't this page need the root HTML content?

Reply:
Note that I'd prefer not to add a separate razor page layout to the template, as that will likely cause issues with scaffolding. See https://github.com/dotnet/aspnetcore/issues/22712.

Reply:
Good point.

Doesn't that basically invalidate the idea of making this change, then?

Reply:
@SteveSandersonMS not really. We can just put the layout contents within the error page. We don't need a layout for this.

Reply:
That's OK for the template out of the box, but it means that developers now have to duplicate all their work on layouts in two places. If they have parts of the layout that are nontrivial to render (a nested chain of `.razor` components), they have to implement it a second time in `.cshtml` too, so it's potentially a significant usability cost, and one that is not even obvious you have to do, so many devs will not do it and then be confused about why their error pages look all mangled and incorrectly styled once they've edited `.css`.

My question is: is this usability cost worth it for the benefit we're providing here? If our goal is just to fix the infinite loop edge case, is there a different way we can fix that?

@danroth27 It would be good to have your opinion on the usability cost here.

Reply:
Maybe the answer is that, for the error page, we shouldn't try to recreate the same appearance as the main layout. Maybe it should be a totally plain white page with just the error text.

Reply:
Yeah, I was thinking the error page could be simple enough that it's not critical that it picks up the app layout in this case.